contrib/format-esri/src/test/java/org/apache/drill/exec/store/esri/TestShapefileFormatPlugin.java

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
Why you need to add `contrib/format-esri/src/test/resources/shapefiles/CA-cities.parquet` in resources?

Reply:
Removed file.  Not sure why that was there.